pynamodb/connection/base.py

```diff
@@ -427,11 +427,19 @@ def _make_api_call(self, operation_name, operation_kwargs):
                     'request_id': headers.get('x-amzn-RequestId')
                 }
 
-                if 'RequestItems' in operation_kwargs:
+                if REQUEST_ITEMS in operation_kwargs:
                     # Batch operations can hit multiple tables, report them comma separated
-                    verbose_properties['table_name'] = ','.join(operation_kwargs['RequestItems'])
+                    verbose_properties['table_name'] = ','.join(operation_kwargs[REQUEST_ITEMS])
+                elif TRANSACT_ITEMS in operation_kwargs:
+                    # Transactional operations can also hit multiple tables, or have multiple updates within
+                    # the same table
+                    items = []
+                    for item in operation_kwargs[TRANSACT_ITEMS]:
+                        for op in item.keys():
+                            items.append(json.dumps(item[op][TABLE_NAME]))
+                    verbose_properties['table_name'] = ','.join(items)
```

Comment:
Should we also add a `table_names` for when you are aware that you've made a transactional request and you want to avoid parsing a string?

Reply:
Oh, never mind. I assumed the `verbose_properties` somehow become accessible through `VerboseClientError` but those are just used in the message interpolation.